ConfigData/phase GEMS Comparison.py

```diff
@@ -239,19 +249,68 @@ def GEMSChiPlot(AtPct, Protosolar):
     # print('xmean: ', xmean)
     # print('Chi Squared result from scipy:', ScipyChiSquare)
 
-
-    plt.figure(3)
-    plt.clf()
+    # FIXME
+    # TODO Approach that using mpld3, failed.
+    # fig = Figure(figsize=(5,4))
+    # ax = fig.subplots()
+    # IncludedZ = where(~isnan(Chondricity))[0]+1
+    # TickLabels = [El for Z, El in enumerate(pb.ElementalSymbols) if Z in IncludedZ]
+    # TickInds = list(range(len(TickLabels)))
+    # ax.scatter(TickInds, Chondricity[IncludedZ-1], marker='o', color='red', s=150, alpha=0.5, label='Chondricity')
+    # ax.scatter(TickInds, ElementSigma[IncludedZ-1], marker='o', color='blue', s=150, alpha=0.5, label='Elemental sigmas from mean')
+    # ax.set_xticks(TickInds, TickLabels, rotation='vertical')
```

Comment:
If this approach to using mpld3 failed (and it's all commented out) then it shouldn't be committed to the repository.  If you think this is something you think you'll want to pick up again later, then it should be kept in a local branch on your system so you can checkout that branch at a later date and finish it.  Only done stuff goes in the main repository.